pkg/virtualrouter/enqueue_requests_for_mesh_events.go

```diff
@@ -34,7 +34,7 @@ func (h *enqueueRequestsForMeshEvents) Create(e event.CreateEvent, queue workque
 
 // Update is called in response to an update event
 func (h *enqueueRequestsForMeshEvents) Update(e event.UpdateEvent, queue workqueue.RateLimitingInterface) {
-	// virtualNode reconcile depends on mesh is active or not.
+	// virtualRouter reconcile depends on mesh is active or not.
 	// so we only need to trigger virtualNode reconcile if mesh's active status changed.
```

Comment:
Good catch. You might want to change virtualNode->virtualRouter in line 38 as well